### ISSUE TITLE
The training loss is decreasing while the validation loss is increasing

### ISSUE THREAD

**Issue by WhiteLie98** [IcarusWizard/MAE#6]
Sorry to bother you again. I use the pretrain model of yours and run the train_classifier.py. I visualized the log by tensorboard and found the training loss is decreasing while validation loss is increasing. I wonder if the model is overfitting?

**Comment by WhiteLie98** [comment]
![a9c8204d9aaa64259e1d1bc423f8b25](https://user-images.githubusercontent.com/39949809/146715417-65bec24b-f49e-493d-b1e7-efbb86bda265.jpg)
These are the results of logs.

**Comment by IcarusWizard** [comment]
Yes, the model is overfitting since cifar10 is a small dataset and we did not perform any data augmentation.

**Comment by WhiteLie98** [comment]
I see. And where should I extend the dataset by using data augmentations? In the pretrain program or fine-tuning program?
p.s. I'm sorry to ask such questions as this was the first CV program I run.

**Comment by IcarusWizard** [comment]
You can add data augmentations when defining the dataset, i.e. changing the transform from [here](https://github.com/IcarusWizard/MAE/blob/main/train_classifier.py#L35). Something like `RandomResizedCrop` should be very helpful. But unfortunately, I do not have the computation resource to verify.
